osquery/tables/other/yara.cpp

```diff
@@ -147,35 +146,16 @@ QueryData genYara(QueryContext& context) {
       }
 
       VLOG(1) << "Scanning with group: " << group;
-      status = doYARAScan(rules[group],
-                          path_pair.first.c_str(),
-                          path_pair.second,
-                          results,
-                          group,
-                          "");
+      auto status = doYARAScan(rules[group],
+                               path_pair.first.c_str(),
+                               path_pair.second,
+                               results,
+                               group,
+                               group);
```

Comment:
I think this may be wrong, but it's hard to double check on my phone. Shouldn't the last argument be "" since we aren't scanning with a sigfile, but rather a group?